databricks/koalas/frame.py

```diff
@@ -3097,6 +3097,84 @@ def pandas_between_time(pdf) -> ks.DataFrame[return_types]:  # type: ignore
             )
         )
 
+    # TODO: implement axis=1
+    def at_time(
+        self, time: Union[datetime.time, str], asof: bool = False, axis: Union[int, str] = 0
+    ):
+        """
+        Select values at particular time of day (e.g., 9:30AM).
+
+        Parameters
+        ----------
+        time : datetime.time or str
+        axis : {0 or 'index', 1 or 'columns'}, default 0
+
+            .. versionadded:: 0.24.0
+
+        Returns
+        -------
+        Series or DataFrame
```

Comment:
Do we have a case to return `Series`?

Reply:
Also seems like `between_time` has the same issue.

Reply:
Makes sense!